apcd-cms/src/apps/admin_exception/templates/view_admin_exception_modal.html

```diff
@@ -1,65 +1,11 @@
+{% extends "apcd_cms/templates/standard.html" %}
 {% load static %}
 <link rel="stylesheet" href="{% static 'apcd-cms/css/modal.css' %}">
```

Comment:
Some of your style modules (ViewExceptionModal.module.css in this case) may be clashing with these stylesheets here and might be why the modals don't look the same as dev/staging/prod.